unitTests/curvatureResistance/curvatureResistance.py

```diff
@@ -0,0 +1,285 @@
+import json
+import matplotlib.pyplot as plt
+import numpy as np
+import os
+import sys
+import tempfile
+import unittest
+
+sys.path.append('../..')
+
+from efficiency import totalLossesFunction
+from ocp import casadiSolver
+from train import Train
+from track import Track
+from utils import postProcessDataFrame
+
+
+class TestCurvatureResistance(unittest.TestCase):
+
+    TRACK = "../../tracks/00_stationX_stationY.json"
+
+    CONSTANT_K = 1/300 # [m^-1]
+
+    DEFAULT_TRIP_FINAL_POSITION = 3475 # [m]
+
+    DEFAULT_SPEED_AT_START = 1 # [m/s]
+
+    DEFAULT_SPEED_AT_END = 1 # [m/s]
+    
+    THRESHOLD_VELOCITY_ERROR_MINIMUM_TIME_PROBLEM = 1e-3 # adimensional quantity
+
+    THRESHOLD_ENERGY_ERROR_MINIMUM_ENERGY_PROBLEM = 5e-2 # adimensional quantity
+
+    @classmethod
+    def setUpClass(cls):
+        '''
+        Based on the dataset contained in TRACK, two temporary track datasets are created.
```

Comment:
Isn't it simpler to create two track objects from the existing json file instead of creating temporary json files?